save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ProjectController.kt

```diff
@@ -62,7 +63,11 @@ class ProjectController(
         description = "Get all projects, including deleted and private. Only accessible for super admins",
     )
     @ApiResponse(responseCode = "200", description = "Projects successfully fetched.")
-    fun getProjects(): Flux<Project> = projectService.getProjects()
+    fun getProjectsWithStatus(@RequestParam status: ProjectStatus?): Flux<Project> =
+            projectService.getProjects()
+                .filter { project ->
+                    status?.let { project.status == it } ?: true
+                }
```

Comment:
Filtering should be done with database request

Reply:
Is it adequate to do this not at the database request level, but in the Service